### PR TITLE
Replaced WebL10n with i18next in FoodChain activity

### PR DESCRIPTION
**Pull request by soham2312**
fix issue #1378 

Replaced Code in FoodChain Activity to l10n

@llaske , could you please review it

### CODE REVIEW

**Comment by llaske** [comment]
There is an error on launch of the activity 

image

File: activities/FoodChain.activity/lib/sugar-web/activity/activity.js
Comment (Vishalk91-4) [inline activities/FoodChain.activity/lib/sugar-web/activity/activity.js:1]: remove l10n in the function( 

You have removed webL10n but not it's corresponding l10n

**Review by Vishalk91-4 — CHANGES_REQUESTED** [review]

**Comment by soham2312** [comment]
@llaske sir can you review it

**Comment by llaske** [comment]
Nice @soham2312, thanks

**Pull request merged**